dispatch/tests/test_api_files.py

```diff
@@ -0,0 +1,93 @@
+import os
+import shutil
+import datetime
+from django.core.urlresolvers import reverse
+from rest_framework import status
+from dispatch.tests.cases import DispatchAPITestCase
+
+
+class FileTests(DispatchAPITestCase):
+
+    def _upload_file(self):
+        """
+        Upload a test file... somehow
+        """
+        url = reverse('api-files-list')
+
+        with open(os.path.join(os.path.dirname(__file__),'input/TestFile.txt')) as fp:
```

Comment:
You can also remove the need for an external file by using [`stringio`](https://docs.python.org/2/library/stringio.html), a builtin python library that creates file objects in memory:
```py
import StringIO
...

test_file = StringIO.StringIO('testtesttest')
data = {
    'name': 'TestFile',
    'file': test_file
}

...

test_file.close()